### DISCUSSION TITLE
Navyfication of NSV (Or how to make the game actually about warships)

### DISCUSSION THREAD

**Discussion by Absolutely-Steph** [BeeStation/NSV13#1578] [Ideas]
### This is the main point to take away from this. It was proven correct by Midshipmen bettering their behaviour.
this is gonna be a long one, sorry.
### **My perceived problem:**
SS13 is clashing with NSV13 in a really ugly way, nonmuni roles rarely have any difference to playing standard ss13 but with shaking and loud sounds.
Excerpt from the review "some rogue agent" posted on [reddit](https://www.reddit.com/r/SS13/comments/on28rh/the_guide_to_unique_servers_episode_5_nsv13/): 

> Once you step away from vessel combat, it quickly becomes very clear that it was not just the main focus of the developers - it was the only focus. The rest of the roles are very basic. Almost none of them have gotten a rehaul to adjust to the realities of void warfare.
You’ve got your botanist, virologist, lawyer, mime, etc. What are they doing on the warship in the middle of active combat? I have no clue. And neither does the development team it seems.
On a regular station, these roles offer different spins on the round. Even the most boring and memey of them could be used in an interesting way to participate in the action fighting antagonists or contributing in some other way. In this case, they are just low-tier roles that have very little connection to what is actually happening.

**I am in favour of removing all non-overmap roles. That may be too extreme to accept, so here is a compromise**

### We have already solved this problem with a single role, albeit unintentionally

**Before boarding, assistants were greytiders like on any other servers, players like jeffery made up a good amount of assistant, the others mostly walking around learning the game or without concrete focus.
Today, midshipmen are a reliable source of gunners and field repair specialists, they actually mostly listen to what the squad pager tells them to do. Simply because their name change brought with it a different association.**

I propose that we change as much as we can to split off from regular station gameplay, In hopes of involving the entire crew with overmap combat. This can be done in a multitude of ways.
The easiest of which is detailed below:

### Don't change the game, Allow players to change themselves instead.
**Many players are interested in heavier-ish roleplay, Most others wouldn't mind it.**
I say we can probably crank it up a bit, starting with immersion and psychology. make it FEEL like you are on a warship, they will gladly take your hand and follow along.
**Getting the feel right is not a matter of overhauling large swaths of gameplay**, we can work on the details and the rest will follow naturally. This is what I think our first steps should be:

### Step 1: Get rid of roundstart individuality, why does the cook look like a damn restaurant chef from the 60ties.
**Everyone gets a standard serviceman uniform. Officers Included**
No fancy roundstart parade jackets(looking at you MAA). Officers get officers dress (XO standard for example), servicemen get standard uniforms(Midshipmen uniforms?).
From there, players can decorate themselves as much as they want. Engineers can just throw on a hazard vest so that people recognize them, **most PPE already organically identifies departments, Labcoats, Munihelmets, Hazard vests, Galoshes, Aprons, etc.**

That should already bring people into the mindset that they are NAVY PERSONNEL. **We can go even further though:**
### Step 2: Rename EVERY JOB(if needed)
This is the big kicker(and depending on how well the uniforms work, might not be needed), if people go in with the expectation that they will be a botanist, they will just sit there and water their plants, no matter what happens to the ship.
Now rename it to "Hydroponics Specialist" (Name placeholder, im not good with names) and they will already behave more professionally by simple association. **No gameplay changes, no balancing, no headache(except renaming woops).**

### TLDR
**Read it you slacker, but in essence; Make roles on a warship feel like they belong there, no civilians are supposed to be there.
NO ROLES WILL BE REMOVED, NO ONE GETS FORCED TO ACT DIFFERENTLY, EVEN IF THEY DO NOT CHANGE THEIR BEHAVIOUR THE NEW NAMES AND UNIFORMS WILL BE MORE IMMERSIVE ANYWAYS WHICH IS WORTH IT.**
